zorg/buildbot/builders/FlangBuilder.py

```diff
@@ -16,11 +16,14 @@ def getFlangOutOfTreeBuildFactory(
     if env is None:
         env = dict()
 
+    install_dir = "install"
```

Comment:
I assume that by passing install_dir to the factory function below, that generates an install step. If you leave it out, it does not.

Can you put a comment here that links to the issue we're working around?

Reply:
I'd also add the info that the `getCmakeWithNinjaBuildFactory` and `addNinjaSteps` later are installing the files into a common install_dir/prefix combining clang's/flang's resource directory